src/main/java/scala_maven_dependency/ArtifactIds4Scala3.java

```diff
@@ -0,0 +1,70 @@
+
+/*
+ * Copyright 2011-2020 scala-maven-plugin project (https://davidb.github.io/scala-maven-plugin/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package scala_maven_dependency;
+
+import java.util.Arrays;
+import java.util.List;
+import scala_maven.VersionNumber;
+
+public class ArtifactIds4Scala3 implements ArtifactIds {
+  protected static final String SCALA_LIBRARY_ARTIFACTID = "scala3-library";
+  protected static final String SCALA_INTERFACES_ARTIFACTID = "scala3-interfaces";
+  protected static final String SCALA_REFLECT_ARTIFACTID = "scala3-reflect";
```

Comment:
scala3 doesn't have reflect jar. may be we should remove this.

Reply:
thanks, I refactored and was able to also remove it from scala2